docs/validator/economics.md

```diff
@@ -0,0 +1,132 @@
+In practical terms, you have to:
+- Re-stake your `proposal` again before the end of the failed epoch, and (if your `proposal` succeed),
+- Wait for one more `epoch`, which is needed to sync all validators with their newly assigned shard
+
+With 730 epochs per year, you will miss at least 24 hours of rewards - therefore plan redundancy over several locations, with eventual connectivity fallback.
+
+### Consistency
+
+Each validator must submit a *valid state* to the blockchain, or the stake will get slashed. Beware that validators may be slashed if any fallback systems intervene by mistake, generating blocks with the same key across different nodes. Therefore, as a validator, you need a reliable solution that maintains consistency across different datacenters and geographies.
+There are many more slashing conditions, with documentation that is work in progress, please refer to the [EpochManager specs](https://github.com/nearprotocol/NEPs/pull/37) if you want to know more.
+
+
+## 3. Continuous operation
+
+Validators must keep the operations up and running by:
+1. Setting the *Service Level Objectives*
+2. Leveraging market dynamics
+
+### Setting the Service Level Objectives
+
+Achieving 90% of uptime is not sufficient: any new release of the node should be tested and applied without interruptions. Leveraging infrastructure-as-code usually simplifies ongoing maintenance, increasing efficiency and profitability at the cost of higher upfront investments.
+On top of ordinary maintenance, your operations will require an incident/response playbook, to promptly resolve any infrastructure failure, as required by the `MINIMUM_THRESHOLD`.
+
+
+### Leverage market dynamics
+
+NEAR validators are subject to market and game-theory dynamics, relating to:
+1. The total of NEAR tokens at stake
+2. The median of the stake across validators
+3. Protocol fees and inflation
+4. The exchange value of NEAR tokens
+5. The opportunity cost of other PoS tokens
+
+**Total of NEAR tokens at stake**
+NEAR rewards algorithm mint 5% new tokens every year. Therefore, the initial supply of 1 billion NEAR will generate 50 million tokens (5% of 1 billion) in stake rewards.
+This reward is fixed, regardless of the stake:
+- A relatively small stake of 100 million (10% of 1 billion) will generate a nominal return of 50% (50 million NEAR for a stake of 100 million), thus attracting more staking.
+- On the other hand, a large stake of 800 million (80% of 1 billion) will return 6.25% in rewards (50 million NEAR for a stake of 800 million), which may become too low to lock so many tokens.
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    The protocol automatically re-stakes your rewards at the end of each epoch, so you may expect up to 5% of additional stake every year.
+</blockquote>
+
+**Median of the stake across validators**
```

Comment:
so, to me median is a confusing metric.

Because you can have: [1,000,000, 1, 1, 1, 1] -> the first proposal will take all 100 seats, even though median is 1.

I think we should show here an example, with proposals, how to find the seat price and number of seats. Also describe how rewards are still proportional to the stake vs # seats.

Something like, given proposals (with rollovers from previous epoch):
| Validator | Stake | # Seats |
| - | - | - |
V1 |   1,000,000    | 48
V2 |      500,000   | 24
V3 |      300,000   | 14 
V4 |      300,000 | 14
--- | seat price is 20,500 |  --- |
V5 |         20,000 | 0 |

Where seat price is such an integer number, such that if you divide all proposals and round down by it - the number of seats will be greater or equal than required.

Reply:
My problem is explaining how `argmax` is applied to this process. Working on it 🤔